keep/providers/kubernetes_provider/kubernetes_provider.py

```diff
@@ -0,0 +1,203 @@
+import pydantic
+import dataclasses
+
+from kubernetes import client
```

Comment:
add dependency to pyproject

Reply:
https://github.com/keephq/keep/blob/main/pyproject.toml#L58 it exists already 🙂